build.gradle

```diff
@@ -386,6 +397,7 @@ ext.versions = [
   hamcrest            : '1.3',
   jetty               : '9.2.18.v20160721',
   jgit                : '4.9.0.201710071750-r',
+  jruby               : '9.1.14.0',
```

Comment:
@ketan - [`9.1.15.0`](https://github.com/jruby/jruby/releases/tag/9.1.15.0) of jruby is out. can we upgrade to `9.1.15`?

Reply:
@GaneshSPatil — I tried and rolled back. See https://github.com/jruby/jruby/issues/4858#issuecomment-352851054